### ISSUE TITLE
When running multiple haproxy containers on Tutum Cloud, login is presented

### ISSUE THREAD

**Issue by willrstern** [tutumcloud/haproxy#122]
It only happens when using more than one `target_num_contaners`:
- create a hello-world service
- link a 2-node haproxy lb
- one or more lb's endpoints will ask for a login
![image](https://cloud.githubusercontent.com/assets/1680215/11343726/f2df5f8c-91d2-11e5-9c22-4414cc176a9b.png)
- note, it happens more often than not, although, if you change scale, you can sometimes get all lb's working/not asking for `haproxy statistics` login

A sample config:
```
haproxy:
  image: 'tutum/haproxy:latest'
  deployment_strategy: high_availability
  links:
    - web
  ports:
    - '80:80'
  restart: always
  sequential_deployment: true
  target_num_containers: 2
web:
  image: 'tutum/hello-world:latest'
  command: '/bin/sh -c ''php-fpm -d variables_order="EGPCS" && exec nginx -g "daemon off;"'' '
  deployment_strategy: high_availability
  expose:
    - '80'
  restart: always
  sequential_deployment: true
  target_num_containers: 4
```

**Comment by tifayuki** [comment]
Two problems with your stackfile:

1. You should always give `global` role to haproxy container, with which haproxy can configure itself via tutum api dynamically. Otherwise, the container only is configured once base on the environment variables.

2. DO NOT use sequential_deployment. Sequential_deployment is not designed for such a scenario. Sequential_deployment should only be used when a following container relies information on the previous container in the same service. Take mysql service with master/slave configuration as an example, the second slave container need to know the information of the first master container. To make it possible, tutum adds the environment of the first container to the seconds one.

To sum it up, with your configuration, the first haproxy appears to be "linked" to the second one, including the stats route of the first container.
